website/docs/profile.md

```diff
@@ -0,0 +1,28 @@
+Enable or disable instrumented performance profiling support for binaries.
+
+```lua
+profile "value"
+```
+
+### Parameters ###
+| Value   | Description                                                             |
+-------------------------------------------------------------------------------------
+| Default | Use the toolset's default instrumentated performance profiling setting. |
```

Comment:
Does the boolean kind support default? I thought it was a hack we achieved via the string kind.